web-mode/vue

```diff
@@ -0,0 +1,18 @@
+# -*- mode: snippet -*-
+# name: vue
+# uuid: vue
+# key: vue
+# condition: t
```

Comment:
Change condition from `t` to `(equal web-mode-engine "vue")`. `condition` controls when the snippet should be shown.